.ycm_extra_conf.py

```diff
@@ -1,24 +1,20 @@
-import os
-import sys
-
+import sysconfig
 from pathlib import Path
 
-CONDA_PREFIX = os.environ["CONDA_PREFIX"]
+PREFIX = sysconfig.get_config_var("prefix")
 NVJITLINK_DIR = Path(__file__).parent
-VER = sys.version_info
-PYTHON_INCLUDE_NAME = "python3.10"  # Not ideal
 
-CONDA_INCLUDE_DIR = Path(CONDA_PREFIX, "include")
-PYTHON_INCLUDE_DIR = Path(CONDA_INCLUDE_DIR, PYTHON_INCLUDE_NAME)
+PREFIX_INCLUDE_DIR = Path(PREFIX, "include")
+PYTHON_INCLUDE_DIR = sysconfig.get_config_var("include")
 NVJITLINK_INCLUDE_DIR = Path(NVJITLINK_DIR, "include")
-CUDA_INCLUDE_DIR = "/usr/local/cuda-12.0/include"
+CUDA_INCLUDE_DIR = "/usr/local/cuda/include"
 
 flags = [
     "--cuda-gpu-arch=sm_50",
```

Comment:
RAPIDS requires arch 70 or higher.
```suggestion
    "--cuda-gpu-arch=sm_70",
```

Reply:
Could you please submit a new issue or PR?